scripts/SREP/mg-log-extractor/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+file: script.sh
+name: mg-log-extractor
+description: |
+  Extracts must-gether logs from hive s3 bucket
+  ocm backplane mg-log-extractor <s3-file-name> <clustername>-must-gather.tar.gz
+  tar -xvf <clustername>-must-gather.tar.gz
+author: tfahlman
+allowedGroups: 
+  - SREP
```

Comment:
Do we want CEE use this script?

Reply:
No. They don't have access to hives anyway I think